### ISSUE TITLE
Exclude "Done" cards from upcoming view

### ISSUE THREAD

**Issue by varac** [nextcloud/deck#2449]
### Is your feature request related to a problem? Please describe.

I really like the "upcoming cards" view - however when you move a card to a "Done" column they still show up in the upcoming cards view, which is counter-intuitive to what this view is meant.
This can be solved by archiving "Done" cards but with a lot of boards this task is tedious.

### Describe the solution you'd like

Cards from lists called "Done" should be excluded from the upcoming cards view.

### Describe alternatives you've considered

None.

**Comment by armaccloud** [comment]
I think this is a good idea, however I'd rather make it an indicator of some sort instead of using the name of a list.
This is because others would want the same functionality in lists called _completed, finished, waiting for, in queue,_ etc.

**Comment by juliushaertl** [comment]
I'll close this as a duplicate of https://github.com/nextcloud/deck/issues/534 since we need a done state on the cards first before we can somehow hide cards from the view based on that.
